io.github.ecotubehq.player.json

```diff
@@ -0,0 +1,360 @@
+{
+    "app-id": "io.github.ecotubehq.player",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "command": "ecotube",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=all",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-pictures",
+        "--talk-name=org.gtk.vfs.*",
+        "--env=LC_NUMERIC=C"
```

Comment:
you shouldn't set this to "C". Localization is user specific. Not everybody want numbers formatted like Americans do.